### PR TITLE
Move QoS configuration to config DB

### PR DESCRIPTION
**Pull request by andriymoroz-mlnx**
**What I did**
Move QoS configuration to config DB

See also:
https://github.com/Azure/sonic-swss-common/pull/158
https://github.com/Azure/sonic-buildimage/pull/1162

### CODE REVIEW

**Comment by andriymoroz-mlnx** [comment]
> FOR REVIEW AND DISCUSSION
> PLEASE DO NOT MERGE

qui

**Comment by yxieca** [comment]
retest this please

File: orchagent/qosorch.cpp [hunk 1]
```diff
@@ -1014,7 +1014,7 @@ task_process_status QosOrch::handleQueueTable(Consumer& consumer)
 
     ref_resolve_status  resolve_result;
     // sample "QUEUE_TABLE:ETHERNET4:1"
-    tokens = tokenize(key, delimiter);
+    tokens = tokenize(key, delimiter2);
```
Comment (yxieca) [inline orchagent/qosorch.cpp]: The example doesn't match the code change here. Can you also update the example?

File: orchagent/orch.h [hunk 2]
```diff
@@ -18,6 +18,7 @@ using namespace std;
 using namespace swss;
 
 const char delimiter           = ':';
+const char delimiter2          = '|';
```
Comment (yxieca) [inline orchagent/orch.h]: delimiter2 doesn't tell what it is separating, can you give it a more meaningful name? like field_delimiter if it is separating fields, or attr_delimiter if it is separating attributes, for example.

**Comment by yxieca** [comment]
retest this please

**Comment by yxieca** [comment]
retest this please

**Pull request merged**